tables/alb_access_log/alb_access_log.go

```diff
@@ -0,0 +1,222 @@
+package alb_access_log
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/turbot/tailpipe-plugin-sdk/schema"
+)
+
+type AlbAccessLogBatch struct {
+	Records []AlbAccessLog `json:"Records"`
+}
```

Comment:
Is this used for anything?
Table sources are all set with `RowPerLine` as such we'll be processing 1 line as 1 item.

Reply:
No, removing it